src/bin/languageserver/mod.rs

```diff
@@ -1,5 +1,8 @@
 // SPDX-License-Identifier: Apache-2.0
 
+#![allow(unstable_name_collisions)]
```

Comment:
I'd prefer to use the fully qualified syntax `itertools::Itertools::intersperse(...)` instead of disabling the lint (as clippy suggests)